src/main/java/org/junit/rules/ExpectedException.java

```diff
@@ -14,7 +14,10 @@
 
 /**
  * The {@code ExpectedException} rule allows you to verify that your code
- * throws a specific exception.
+ * throws a specific exception. Note that {@link org.junit.Assert#assertThrows Assert.assertThrows}
```

Comment:
I new wording looks good, but probably belongs in aa new paragraph, probably near the bottom.of the Javadoc.

I am also not sure if the link tags will work if you don't specify parameters.

Finally, I think you can remove the parenthesis around the last sentence.